core/chains/evm/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -108,32 +108,39 @@ func (b *BulletproofTxManager) RegisterResumeCallback(fn ResumeCallback) {
 }
 
 // NewBulletproofTxManager creates a new BulletproofTxManager with the given configuration.
-func NewBulletproofTxManager(db *sqlx.DB, ethClient evmclient.Client, config Config, keyStore KeyStore, eventBroadcaster pg.EventBroadcaster, lggr logger.Logger, checkerFactory TransmitCheckerFactory) *BulletproofTxManager {
+func NewBulletproofTxManager(db *sqlx.DB, ethClient evmclient.Client, cfg Config, keyStore KeyStore, eventBroadcaster pg.EventBroadcaster, lggr logger.Logger, checkerFactory TransmitCheckerFactory) *BulletproofTxManager {
 	lggr = lggr.Named("BulletproofTxManager")
+	lggr.Infow("Initializing EVM transaction manager",
```

Comment:
Not sure if relevant, but is it helpful/possible to also log the chainId here?

Reply:
It automatically does that since chain ID is already on the logger